content/es/es6.md

```diff
@@ -51,10 +53,21 @@ Puedes ver una lista más detallada, incluyendo la comparación con otros motore
 
 ## ¿Qué funcionales de ES6 están detrás de la --es_staging flag?
 
-* [Clases](https://github.com/lukehoban/es6features#classes) (sólo en strict mode)
-* [Object literal extensions](https://github.com/lukehoban/es6features#enhanced-object-literals)
+* [Clases](https://github.com/lukehoban/es6features#classes) (sólo en strict mode, utilizando la flag `--harmony_classes` lo que implica que el block scope y object literal extensions)
```

Comment:
Ok, perfect. I would also drop the article for "flag" as you did for "el block" -> "block". And yes, I'll replace it anywhere. I always grep the document for laziness anyway :D, but thanks.